mouse/record.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import print_function
+
+__author__ = "Bailey Nozomu Hwa <hwab@janelia.hhmi.org"
+__date__ = "$July 30, 2015 20:20:18 EDT$"
+
+import argparse
+import io
+import picamera
+import datetime
+import os
+
+
+def main(*argv):
+    """
+        Simple main function that performs event center recording.
+        Defines arguments:
+
+        Args:
+            argv(str):                         Arguments are stored as a list
+            folder(str):                      File directory
```

Comment:
Dear John,
Where should I put the notes section?
Bailey




From: jakirkham <notifications@github.com>
Reply-To: DudLab/mouse_record <reply@reply.github.com>
Date: Wednesday, August 5, 2015 at 12:35 PM
To: DudLab/mouse_record <mouse_record@noreply.github.com>
Cc: Hwa Bailey <hwab@janelia.hhmi.org>
Subject: Re: [mouse_record] Complete package (w/o tests) (#1)





In 
mouse/record.py:
> +
> +import argparse
> +import io
> +import picamera
> +import datetime
> +import os
> +
> +
> +def main(*argv):
> +    """
> +        Simple main function that performs event center recording.
> +        Defines arguments:
> +
> +        Args:
> +            argv(str):                         Arguments are stored as a list
> +            folder(str):                      File directory

Move folder, x, and y to new Notes: section.
�
Reply to this email directly or 
view it on GitHub.